juriscraper/opinions/united_states/state/ala.py

```diff
@@ -0,0 +1,98 @@
+"""
+Scraper for Alabama Supreme Court
+CourtID: ala
+Court Short Name: Ala.
+Author: William E. Palin
+Court Contact:
+Reviewer:
+History:
+ - 2021-12-19: Created.
+"""
+from typing import Dict
+
+import pdfplumber
+
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.url = (
+            "https://judicial.alabama.gov/decision/supremecourtdecisions"
+        )
+        self.court_id = self.__module__
+        self.pdf = None
+
+    def _download(self, request_dict={}):
```

Comment:
This is another one that's abusing the download method. I think we need to avoid this.